src/models/installation.ts

```diff
@@ -84,7 +84,8 @@ export class Installation extends EncryptedModel {
 			await installation
 				.update({
 					encryptedSharedSecret: payload.sharedSecret,
-					jiraHost: payload.host
+					jiraHost: payload.host,
+					plainClientKey: payload.clientKey
```

Comment:
Let me know whether setting the `plainClientKey` make sense here.

Reply:
Hmm, that's the site rename, right? Feels like we are solving 2 problems here: doing the update and populating the plainClientKey. I'd suggest not to do that:
 - that doesn't solve much of a problem cause site rename is a rare operation
 - we are making the code more complicated and breaking single responsibility principle.

However, not a strong opinion :)

Reply:
Totally agree, and "plainClientKey" is a primary-key-ish stuff, we shouldn't update it at all. Was thinking only temporary doing this for the life-cycle-update events.

But I've updated the code to update null plain client key in the install event itself. Can you please review?

Reply:
Okay I've put it back @mboudreau :)